### DISCUSSION TITLE
subtype of transform

### DISCUSSION THREAD

**Discussion by slee-prenuvo** [ANTsX/ANTs#1759] [Q&A]
Hello. I am using antspy and have a question regarding antsRegistrationSyN.
In the ants registration script, it says that "x is one of the transforms available (e.g., 't', 'b', 's')" for antsRegistrationSyN[x]. What do the 't', 'b', and 's' mean? Also, what are the options available other than 't', 'b', and 's'?
Thank you!

**Comment by ntustison** [comment]
https://github.com/ANTsX/ANTs/blob/master/Scripts/antsRegistrationSyN.sh#L136-L145
